docs/vNext/Ethereum_Configuration.md

```diff
@@ -142,31 +144,56 @@ This configuration takes priority over `fromAddressPassword`.
 "fromAddressPrivateKey": "0x45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"
 ```
 
-## Benchmark address password
+## Benchmark Address Password
 
 The password to use to unlock [benchmark address](#benchmark-address). If there isn't an unlock password, this key must be present as empty string. If the benchmark address private key is present this is not used.
 
 ```json
 "fromAddressPassword": "gottacatchemall"
 ```
 
-## Confirmation blocks
+> Hyperledger Besu does not provide wallet services, so the `fromAddressPassword` option is not supported and the private key variant must be used.
+
+## Confirmation Blocks
 
 It is the number of blocks the adapter will wait before warn Caliper that a transaction has been successfully executed on the network. You can freely tune it from 1 to the desired confirmations. Keep in mind that in the Ethereum main net (PoW), 12 to 20 confirmations can be required to consider a transaction as accepted in the blockchain. If you're using different consensus algorith (like clique in the example network provided) it can be safely brought to a lower value. In any case it is up to you.
 
 ```json
 "transactionConfirmationBlocks": 12
 ```
 
-## Contract configuration
-It is the list, provided as a json object, of contracts to deploy on the network before running the benchmark. You should provide a json entry for each contract; the key will represent the contract identifier to invoke methods on that contract.
+## Contract Configuration
+It is the list, provided as a JSON object, of pre-deployed contracts or contracts to deploy on the network before running the benchmark. You should provide a JSON entry for each contract. The key will represent the contract identifier to invoke methods on that contract.
+
+Depending on whether you plan to use pre-deployed contracts or deploy them during the benchmark, the configuration will differ slightly. 
 
-For each key you must provide a JSON object with the `path` field pointing to the [contract definition file](#contract-definition-file).
+> Defining configurations simultaneously for both pre-deployed contracts and contracts to be deployed by Caliper is currently **not supported** and may result in unexpected errors. This is because opting for pre-deployed contracts means the contract installation phase will be skipped.
```

Comment:
For the contract configuration, we still want to refer to the path property. It's required if you want to install the smart contract but not required if the contract is already deployed. This unfortunately was the only reference to path outside of the example

Reply:
there's also a reference to path in line 195, in the appropriate _Contracts to deploy_ section. Isn't that enough?

Reply:
yep, must have missed that in the PR, sorry